src/Core/Silk.NET.SilkTouch/NativeApiGenerator.cs

```diff
@@ -986,9 +1002,10 @@ private static string GetCallingConvention(CallingConvention convention)
             => convention switch
             {
                 // CallingConvention.Winapi => "", netstandard2.0 doesn't allow this
```

Comment:
Don't forget to do a manual reimplementation of this at some point in the near future!